google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -292,6 +292,18 @@ TEST_F(PgDataTypeIntegrationTest, WriteReadNumeric) {
   EXPECT_THAT(result, IsOkAndHolds(UnorderedElementsAreArray(data)));
 }
 
+TEST_F(PgDataTypeIntegrationTest, WriteReadPgOid) {
+  if (UsingEmulator()) GTEST_SKIP() << "emulator does not support PG.OID";
+
+  std::vector<PgNumeric> const data = {
+      PgOid("0"),   //
+      PgOid("42"),  //
+      PgOid("999"), //
```

Comment:
Run the `checkers` build to update the formatting.

Reply:
Done